tests/test_ops/test_onnx.py

```diff
@@ -58,6 +59,80 @@ def test_nms():
     assert np.allclose(pytorch_score, onnx_score, atol=1e-3)
 
 
+def test_softnms():
+    if not torch.cuda.is_available():
+        warnings.warn('test_softnms should be ran only if cuda is available.')
+        return
+    from packaging import version
+    from mmcv.ops import soft_nms, get_onnxruntime_op_path
+
+    # only support pytorch >= 1.7.0
+    if version.parse(torch.__version__) < version.parse('1.7.0'):
+        warnings.warn('test_softnms should be ran with pytorch >= 1.7.0')
+        return
+
+    # only support onnxruntime >= 1.5.1
```

Comment:
We can just assert the version >= 1.5.1.

Reply:
Only pytorch>=1.7.0 works for soft_nms, which has the `return` value of **a tuple with multiple tensors** in its `forward` function.

Reply:
This is the version of onnxruntime.

Reply:
Ok, warnings would be replaced with assertations.